include/CLI/Timer.hpp

```diff
@@ -10,7 +10,7 @@
 #endif
 
 #include <array>
-#include <chrono>
+#include <chrono> //NOLINT(build/c++11)
```

Comment:
```suggestion
#include <chrono> // NOLINT(build/c++11)
```

While in this case, this file is not added to the single header build, it's best to remain consistent in spacing here since the single header script makes a set out of the includes and will see different spacings and different headers.